e2e_tests/E2ETests.py

```diff
@@ -15,7 +15,7 @@
 import numpy.random as nprdm
 import pandas as pd
 import sqlite3 as sql
-from pandas.util.testing import assert_frame_equal
+from pandas.testing import assert_frame_equal
```

Comment:
Hmmmm, using testing module?

Reply:
What's the future plans for the function?

Reply:
Using the util testing gave an error saying "Use the pandas.testing api version, not this one"
The version update is for the atol/rtol arguments of the function

Reply:
The function was already being used to compare the benchmark and test im_csv files, if that's what you mean?